packages/fxa-payments-server/src/lib/customer.test.ts

```diff
@@ -151,3 +153,18 @@ describe('isExistingStripeCustomer', () => {
     expect(actual).toBe(true);
   });
 });
+
+describe('needsCustomer', () => {
+  it('return true when customer is empty', () => {
+    const actual = needsCustomer(null);
+    expect(actual).toBe(true);
+  });
+  it('return true when does not have payment_provider set', () => {
```

Comment:
This and the next test no longer apply?

Reply:
Whoops, I just forgot to update the test subject, but otherwise it still makes sense.

Instead of checking for `payment_provider` it now checks `customerId`, but otherwise the logic is pretty much the same.